joblog.go

```diff
@@ -28,10 +32,12 @@ type messageLevel struct {
 	Message string `json:"message"`
 }
 
-func New(url, serviceName string, firstMessage interface{}, options ...func(*JobLog)) (jobLog *JobLog) {
+func New(url, serviceName string, jobName string, actionName string, firstMessage interface{}, options ...func(*JobLog)) (jobLog *JobLog) {
 	jobLog = &JobLog{
 		url:         url,
 		serviceName: serviceName,
+		jobName:     jobName,
+		actionName:  actionName,
```

Comment:
You don't need parameters jobName and actionName, you can get it directly from options